horovod/common/common.h

```diff
@@ -86,7 +86,7 @@ class Tensor {
 public:
   virtual const MPIDataType dtype() const = 0;
   virtual const TensorShape shape() const = 0;
-  virtual const char* data() const = 0;
+  virtual const void* data() const = 0;
```

Comment:
Can you align `PersistentBuffer::AccessData` to `const void*` as well?

Reply:
done.